dev/reference/key-monitoring-metrics/pd-dashboard.md

```diff
@@ -18,20 +18,35 @@ category: reference
 - PD role：当前 PD 的角色
 - Storage capacity：TiDB 集群总可用数据库空间大小
 - Current storage size：TiDB 集群目前已用数据库空间大小
-- Number of Regions：当前集群的 Region 总量
-- Leader balance ratio：Leader 数量最多和最少节点相差的百分比，一般小于 5%，节点重启时会有比较大的波动
-- Region balance ratio：Region 数量最多和最少节点相差的百分比，一般小于 5%，新增/下线节点时相差比较大
+- Current storage usage：TiDB 集群存储空间的使用率
 - Normal stores：处于正常状态的节点数目
+- Number of Regions：当前集群的 Region 总量
+- PD scheduler config：PD 调度配置列表
+- Region label isolation level：不同 label 所在的 level 的 Region 数量
+- Label distribution：集群中 TiKV 节点的 label 分布情况
 - Abnormal stores：处于异常状态的节点数目，正常情况应当为 0
-- Current storage usage：TiDB 集群存储空间的使用率
+- pd_cluster_metadata：记录集群 ID，时间戳和生成的 ID
 - Current peer count：当前集群 peer 的总量
-- Metadata information：记录集群 ID，时间戳和生成的 ID
-- Region label isolation level：不同 label 所在的 level 的 Region 数量
 - Region health：每个 Region 的状态，通常情况下，pending 的 peer 应该少于 100，miss 的 peer 不能一直大于 0
 
-![PD Dashboard - Cluster metrics](/media/pd-dashboard-cluster.png)
+![PD Dashboard - Cluster metrics](/media/pd-dashboard-cluster-v2.png)
+
+## Operator
+
+- Schedule operator create：新创建的不同 operator 的数量
+- Schedule operator check：已检查的 operator 的数量，主要检查是否当前步骤已经执行完成，如果是，则执行下一个步骤
+- Schedule operator finish：已完成的 operator 的数量
+- Schedule operator timeout：已超时的 operator 的数量
+- Schedule operator replaced or canceled：已取消或者被替换的 operator 的数量
+- Schedule operators count by state：不同状态的 operator 的数量
+- 99% Operator finish duration：已完成的 operator 中，99% 所需花费的时间
```

Comment:
```suggestion
- 99% Operator finish duration：已完成调度的 operator 中，99% 所需花费的时间
```

Reply:
Please update the description for this section to make it clearer and more friendly to translation.